tsdb/record/record.go

```diff
@@ -37,6 +37,8 @@ const (
 	Samples Type = 2
 	// Tombstones is used to match WAL records of type Tombstones.
 	Tombstones Type = 3
+	// Exemplars is used to match WAL records of type Exemplars.
+	Exemplars Type = 5
```

Comment:
Why are we skipping 4 ?

Reply:
https://github.com/prometheus/prometheus/pull/7771/files#diff-0850da512424792a0e9ca34559be7f3c2e76735f537a48eb6e865f87ea70f457R40-R41 I assumed this would have made it in by now when I updated `record.go`